ui/src/app/views/workspace/component.html

```diff
@@ -0,0 +1,33 @@
+<h2>AoU Research Portal Workspace</h2>
+<div *ngIf="user else loggedout">
+  <p style="color: #333;">Default workspace for July, 2017 demo</p>
+  <div class="area-block">
+    <h3>CDR Version</h3>
+    <div class="indented">
+      <p>CDR version info goes here.</p>
+    </div>
+  </div>
+  <div class="area-block">
+    <h3>Cohorts</h3>
+    <a href="javascript:;">Add a Cohort</a>
```

Comment:
That href is long term going to take you to the page to add a cohort (as specified in slide 6 of the mocks) 
I'm currently adding the todo stubs, which should make things more clear.